make.bat

```diff
@@ -52,10 +54,10 @@ if exist *popcnt.exe del *popcnt.exe
 cd ..
 set include=x86\include\
 CALL:start_timer
-start /min /wait fasmg.exe "x86\fish.asm" "asmFishW_%datestamp%_popcnt.exe" -e 1000 -i "VERSION_OS='W'" -i "PEDANTIC = 1" -i "VERSION_POST = 'popcnt'" %debug%
+start /min fasmg.exe "x86\fish.asm" "asmFishW_%datestamp%_popcnt.exe" -e 1000 -i "VERSION_OS='W'" -i "PEDANTIC = 1" -i "VERSION_POST = 'popcnt'" %debug%
 start /min /wait fasmg.exe "x86\fish.asm" "asmFishW_%datestamp%_bmi2.exe" -e 1000 -i "VERSION_OS='W'" -i "PEDANTIC = 1" -i "VERSION_POST = 'bmi2'" %debug%
-copy asmFishW_%datestamp%_popcnt.exe WindowsOS_binaries
```

Comment:
I prefer "copy" here. I use this all the time for debugging and it would be annoying to have to change directories each time I want to take a bench.

Reply:
What about allBinaries and all? You use `move`, there.

Reply:
Maybe with the following is solved:
`if %debug%==2 copy %filename% %path%
) else (
move %filename% %path%`